eng/common/TestResources/build-test-resource-config.yml

```diff
@@ -61,24 +61,11 @@ steps:
         }
       }
 
+      Write-Host "Merging sub config from files"
```

Comment:
Should we add other helper functions to clean-up this inline code?

Reply:
Yeah I was thinking about just eliminating the inline entirely. I'll go ahead and do that.